app/src/main/java/org/ea/sqrl/activites/MainActivity.java

```diff
@@ -79,7 +79,7 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         IntentResult result = IntentIntegrator.parseActivityResult(requestCode, resultCode, data);
         if(result != null) {
             if(result.getContents() == null) {
-                Log.d("SimplifiedActivity", "Cancelled scan");
+                Log.d("MainActivity", "Cancelled scan");
```

Comment:
This log row should use TAG constant as all other log rows :)

Reply:
Yep, I'll fix this right away.